lib/api/decorators/property_schema_representer.rb

```diff
@@ -32,39 +32,31 @@
 
 module API
   module Decorators
-    class AllowedReferenceLinkRepresenter < Roar::Decorator
-      include Roar::JSON::HAL
+    class PropertySchemaRepresenter < ::API::Decorators::Single
+      def initialize(type: nil, name: nil, current_user: nil)
```

Comment:
You could get rid of the line below by using [required named parameters](https://github.com/ruby/ruby/blob/ruby_2_1/NEWS#L16-17) (didn't know that myself):